website/sidebars.json

```diff
@@ -4,9 +4,13 @@
       "en/contributing/install",
       "en/contributing/working",
       "en/contributing/standards",
+      "en/contributing/writing-style-guide",
```

Comment:
I'm not sure "writing" would connote the same thing at a glance - maybe "style-guide" instead?

Reply:
I’d be his it ant to have this just be “style-guide” because that has strong design connotations. With our project living in that space, I think that could be confusing

Reply:
Our standards page is quite short and probably could be even shorter.  I think I'd actually prefer to move this entire topic into the Standards page.  We really only have Naming standards in there now.  That's for for code standards.  Accessibility really should be in it's own page.  And JSS also probably is better suited somewhere else.  We just didn't have a place for it yet.

Reply:
https://microsoft.github.io/fast-dna/docs/en/contributing/standards for reference to our current standards page.

Reply:
Leaving as is for now, will look at consolidating in a different pull request.
Also, I the correct name would be style guide but I was also concerned with confusing that with design style guides, keeping writing-style-guide for now, can fix if we decide to merge.
The main reason I am not taking this feedback now is that it would require a significant change that would need a new doc review and I don't want to hold up the progress from the first round that gets us a lot more than we have now. We might also consider a standards folder with some children.